protocol/package.json

```diff
@@ -38,11 +38,13 @@
     "solidity-coverage": "^0.8.4",
     "ts-node": "^10.9.1",
     "typechain": "^8.1.1",
-    "typescript": "^4.8.4"
+    "typescript": "^4.8.4",
+    "@safe-global/safe-deployments": "^1.26.0"
   },
   "scripts": {
     "compile": "hardhat compile",
     "typechain": "hardhat typechain",
+    "typechain:safe": "ts-node ./scripts/generateTypechainFiles.ts",
```

Comment:
I don't have a strong preference. /protocol/scripts works, but it could also be appropriate in apps/dapp if the generated typechain would exclusively be used in the dapp

Reply:
moved to dapp